engine/src/main/java/org/entando/entando/aps/system/services/guifragment/api/ApiGuiFragmentInterface.java

```diff
@@ -210,15 +205,19 @@ private String getApiResourceUrl(String fragmentCode, String applicationBaseUrl,
 			return null;
 		}
 		StringBuilder stringBuilder = new StringBuilder(applicationBaseUrl);
-		stringBuilder.append("api/rs/").append(langCode).append("/core/guiFragment");//?code=").append(fragmentCode);
-		if (null == mediaType || mediaType.equals(MediaType.APPLICATION_XML_TYPE)) {
+		stringBuilder.append("api/legacy/").append(langCode).append("/core/guiFragment");
```

Comment:
i'm really afraid to put this a String in the code ... can we use a common constant shared with the controller configuration